### DISCUSSION TITLE
WARNING: The new version of the model will be updated by safetensor, you may need to download it mannully.

### DISCUSSION THREAD

**Discussion by DarksealStudios** [OpenTalker/SadTalker#443] [Q&A] [answered]
WARNING: The new version of the model will be updated by safetensor, you may need to download it mannully. We run the old version of the checkpoint this time!
{'wav2lip_checkpoint': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\wav2lip.pth', 'audio2pose_checkpoint': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\auido2pose_00140-model.pth', 'audio2exp_checkpoint': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\auido2exp_00300-model.pth', 'free_view_checkpoint': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\facevid2vid_00189-model.pth.tar', 'path_of_net_recon_model': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\epoch_20.pth', 'dir_of_BFM_fitting': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui/extensions/SadTalker/src/config', 'audio2pose_yaml_path': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui/extensions/SadTalker/src/config\\auido2pose.yaml', 'audio2exp_yaml_path': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui/extensions/SadTalker/src/config\\auido2exp.yaml', 'use_safetensor': False, 'mappingnet_checkpoint': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui\\models\\SadTalker\\mapping_00109-model.pth.tar', 'facerender_yaml': 'F:\\Ai\\Automatic1111\\stable-diffusion-webui/extensions/SadTalker/src/config\\facerender_still.yaml'}

I get a lot of errors. If I "download manually" should I just get the new files and put them in the same folder or do I need to delete or rename anything?

**Comment by vinthony** [comment, marked as answer]
just get the new files in the same folder is ok.

**Reply by DarksealStudios** [reply]
Edit: Do I just make a new folder "checkpoints" inside sadtalker main folder in extension and place the safetensors there?

Do you know where the new models are? nevermind, I see it, ty!: https://github.com/OpenTalker/SadTalker/releases

**Comment by zstart11** [comment]
我的也是这个问题，请问怎么解决的？
